event_creation/submission/transfer_inputs/behavioral_inputs.yml

```diff
@@ -550,3 +550,40 @@ files:
     origin_directory: *BEH_EXP_DIR
     origin_file: all_items.txt
     destination: experiment_files/wordpool.txt
+
+## EFRCourier
+
+  - name: elemem_files
```

Comment:
I think all of these changes to the behavioral inputs, except maybe the wordpool, are probably redundant. I suspect event_creation would run without this duplication. If you tried and it failed without these additions because of the combination of "groups" then I stand corrected.

Reply:
You were right! All deleted now other than wordpool.